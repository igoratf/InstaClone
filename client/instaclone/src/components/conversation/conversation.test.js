import React from 'react';
import { shallow } from 'enzyme';
import Conversation from './conversation.component';

describe('Conversation component', () => {
  it('should render Conversation component', () => {
    expect(shallow(<Conversation />)).toMatchSnapshot();
  })
})