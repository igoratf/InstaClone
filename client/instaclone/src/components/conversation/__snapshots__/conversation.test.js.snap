// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Conversation component should render Conversation component 1`] = `ShallowWrapper {}`;
